src/journal/journald-context.c

```diff
@@ -784,3 +848,17 @@ void client_context_acquire_default(Server *s) {
 
         }
 }
+
+bool is_log_discarded(ClientContext *c, const char *message) {
```

Comment:
this should probably move to journald-client.[ch]. And be named `client_context_run_message_regex()` or so.